### DISCUSSION TITLE
Главная ⚡️ Angular с примерами кода

### DISCUSSION THREAD

**Discussion by giscus** [xsltdev/angdev.ru#44] [Announcements]
# Главная ⚡️ Angular с примерами кода

Angular представляет собой JavaScript-фреймворк, который позволяет создавать высокопроизводительные и функциональные клиентские web-приложения

https://angdev.ru/

**Comment by margarita-sh** [comment]
Хотя эта документация где-то устарела, уже Angular 16, но лично для меня она до сих пор остается самой сильной. Я всегда перед интрвью я ее читаю и интерью проходят успешно. Русской альтернативы просто нет! С вами уже несколько лет! Спасибо за труд!

**Reply by bndby** [reply]
16 ангуляр в процессе перевода уже, думаю, через пару недель анонсирую на бусти.

**Comment by imvitalya** [comment]
Ребята, начал вкатываться в angular и надеюсь, что ваша дока мне в этом поможет. Хоть и понимаю, что авторская дока не сравнима с оффикал, но все -таки и у меня знание английского не такое идеальное...

Спасибо за ваш труд!

**Comment by Amaratsu** [comment]
Уже 17 вышел

**Comment by Webs95** [comment]
Отличный материал. Большое спаибо, за ваш труд!

**Comment by ramshido** [comment]
Angular => реактивные формы => там где написано [formControlName] ошибка! не работает! сработает только если написать [formControl], если не используются скобки, значит это контрол в FormGroup, исправьте
